test/test_fcntl.rs

```diff
@@ -561,3 +561,21 @@ mod test_posix_fallocate {
         }
     }
 }
+
+#[test]
+#[cfg(any(target_os = "netbsd", target_os = "macos", target_os = "ios"))]
+mod test_apple_netbsd {
+    use nix::fcntl::*;
+    use tempfile::NamedTempFile;
+
+    #[test]
+    fn test_path() {
+        let mut tmp = NamedTempFile::new().unwrap();
+        let fd = tmp.as_raw_fd();
+        let mut path: Vec<u8> = Vec::new();
+        let res = fcntl(fd, FcntlArg::F_GETPATH(path)).expect("get path failed");
+        assert_eq!(
+            path.len() > 0
```

Comment:
This assertion is kinda useless as the length is `libc::PATH_MAX`, which is [definitely](https://docs.rs/libc/latest/x86_64-apple-darwin/libc/constant.PATH_MAX.html) greater than 0